pkg/tiles/side/side.go

```diff
@@ -140,3 +142,73 @@ func (side Side) Rotate(rotations uint) Side {
 	var shift = rotations * 2
 	return (side >> shift) | (side << (8 - shift)) // circular bitshift (bitwise rotate) side to the right by {2*rotations} bits
 }
+
+/*
+argument indicates only ONE cardinal or edge, otherwise it's ambigous
+*/
+func (side Side) ConnectedOpposite() (Side, error) { //nolint:gocyclo // splitting into multiple switches would be obscure
```

Comment:
This function should use bitmask operations, as it will also make nolint directive redundant and would let us  avoid returning error

Reply:
changed